avatica/src/main/java/org/apache/calcite/avatica/proto/Requests.java

```diff
@@ -14,7 +14,6 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-
 // Generated by the protocol buffer compiler.  DO NOT EDIT!
```

Comment:
I thought protobuf compiler could be fetched from maven dependencies in a regular way.
Is it the case? Is the compiler java based or is it a native app?

For instance, we do not keep SqlParserImpl.java under version control.

PS. I don't find relevant mail thread for some reason.